command/state_meta.go

```diff
@@ -23,8 +25,9 @@ func (c *StateMeta) State(m *Meta) (state.State, error) {
 		return nil, err
 	}
 
+	env := c.Env()
```

Comment:
May want to be using `m.Env()` here. Practically I don't think it makes a difference currently but maybe one day we may pass in a differently configured meta here?